modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/latch/ExchangeLatchManagerTest.java

```diff
@@ -16,20 +16,124 @@
 package org.apache.ignite.internal.processors.cache.distributed.dht.preloader.latch;
 
 import java.util.concurrent.TimeUnit;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.configuration.CacheConfiguration;
+import org.apache.ignite.configuration.IgniteConfiguration;
 import org.apache.ignite.internal.GridTopic;
 import org.apache.ignite.internal.IgniteEx;
+import org.apache.ignite.internal.IgniteInternalFuture;
+import org.apache.ignite.internal.TestRecordingCommunicationSpi;
 import org.apache.ignite.internal.managers.communication.GridIoPolicy;
 import org.apache.ignite.internal.processors.affinity.AffinityTopologyVersion;
+import org.apache.ignite.internal.util.typedef.G;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.ListeningTestLogger;
+import org.apache.ignite.testframework.LogListener;
 import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
 import org.junit.Test;
 
 /**
  * Tests for {@link ExchangeLatchManager} functionality when latch coordinator is failed.
  */
 public class ExchangeLatchManagerTest extends GridCommonAbstractTest {
-    /** */
+    /**
+     *
+     */
     private static final String LATCH_NAME = "test";
 
+    /** Message are meaning that node getting a stale acknowledge message. */
+    private static final String STALE_ACK_LOG_MSG = "Ignoring stale latch's acknowledge";
+
+    /** Message happens when assertion was broken. */
+    public static final String ERROR_MSG = "An error occurred processing the message.*LatchAckMessage";
```

Comment:
It looks like a Pattern, please replace it with it, but with the value: "An error occurred processing the message.*LatchAckMessage.*"

Reply:
Thanks, I was fix it.